app/controllers/api/v1/bots_controller.rb

```diff
@@ -109,25 +109,41 @@ def opt_route( @user )
   end
 
   ### JOIN / set user to active
-  def join_route( @user )
-    @user.active = true
-    if @user.save
+  def join( user )
+    user.active = true
+    if user.save
       return "Thanks for joining! :tada:"
     else
       return "Sorry, something went wrong. :robot_face: Please try again."
     end
   end
 
   ### LEAVE / set user to inactive
-  def leave_route( @user )
-    @user.active = false
-    if @user.save
+  def leave( user )
+    user.active = false
+    if user.save
       return "Take a break! :desert_island: Type `opt`, `yes` or `join` anytime to hop back in."
     else
       return "Sorry, something went wrong. :robot_face: Please try again."
     end
   end
 
+  ### ADD QUESTION
+  def save_question( text, user )
+    query = text.gsub("question:", "")
+    query = query.strip
+    if Question.where(question: query).first
+      return "Can you believe......... Someone already asked that!"
+    else
+      question = Question.new(question: query, user: user)
```

Comment:
This works, also, another handle tip can be to do:

```ruby
user.questions.new(question: query)
```

That will automatically set `user_id` on the `Question`